nibabel/cmdline/diff.py

```diff
@@ -101,8 +116,8 @@ def get_headers_diff(file_headers, names=None):
     return difference
 
 
-def get_data_diff(files):
-    """Get difference between md5 values
+def get_data_md5_diff(files):
```

Comment:
Just future-proofing: How about naming it `get_data_hash_diff`? (Understood that the hash will be MD5 for the foreseeable future.)